shell/common/crash_keys.cc

```diff
@@ -93,9 +93,12 @@ static_assert(kMaxCrashKeyNameLength <= crashpad::Annotation::kNameMaxLength,
 void SetCrashKey(const std::string& key, const std::string& value) {
   // Chrome DCHECK()s if we try to set an annotation with a name longer than
   // the max.
-  // TODO(nornagon): warn the developer (via console.warn) when this happens.
-  if (key.size() >= kMaxCrashKeyNameLength)
+  if (key.size() >= kMaxCrashKeyNameLength) {
+    LOG(WARNING) << "Crash key name, \"" << key << "\", is longer than "
```

Comment:
Is there a way to access the `v8::Isolate`/`node::Environment` object inside this gin template function? I could really use some help finding such an example because I can't think of any other way to call `console.warn()` inside this function.

Reply:
Search for `EmitWarning` in the codebase :)

Reply:
But, I think a `LOG` is good enough for now.

Reply:
@nornagon thanks for the hint, I have updated the code to use `EmitWarning()` as I wasn't able to see the output from `LOG()` for some reason but this works as expected now; PTAL :)